### PR TITLE
Add options

### LINKED ISSUE #10: Add option to see animated emojos
There's a lot of animated emojis nowadays and there seems to have no way to see the animated versions in emojos.in, it would be nice to be able to click a button to see animated versions :)

### PR DESCRIPTION
**Pull request by yingziwu**
Add option to see unlisted emojos
Add option to see animated emojos (fix  #10 )

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,133 @@
-__pycache__
+# Created by https://www.gitignore.io/api/python
+# Edit at https://www.gitignore.io/?templates=python
```
Comment (iliana) [inline .gitignore]: Can you drop the .gitignore change from this PR? A significant amount of this is unnecessary.

File: emojos.py [hunk 2]
```diff
@@ -25,13 +25,26 @@
 
 @app.route('/<domain>')
 def emojo(domain):
+    if request.args.get('filter_all','') == 'on':
```
Comment (iliana) [inline emojos.py]: Can you run a style linter? I believe it will require a space after the comma here.

**Review by iliana — CHANGES_REQUESTED** [review]
I'll test this at home tonight, but right now I have a few requests:

(also thank you so much for this!)

**Comment by yingziwu** [comment]
Can you merge this PR?

**Pull request merged**